serialization_benchmarks_test.go

```diff
@@ -131,7 +130,7 @@ func benchUnmarshal(b *testing.B, s Serializer) {
 		o := &A{}
 		err := s.Unmarshal(ser[n], o)
 		if err != nil {
-			b.Fatalf("%s failed to unmarshal: %s (%s)", s, err, ser[n])
```

Comment:
This change seems less helpful, what's the purpose?

Reply:
Ah I see what your goal was, but b.Helper() does not achieve what the old code did.

Reply:
I think the old code had this String description to trace back the respective benchmark. Was there another achievement?

Reply:
Yes it makes it easier to debug when something fails.

Reply:
Printing binaries as UTF seems unwise. It also seems unlikely you'd want to debug in a serialisation benchmark. It is important to report all errors, so one can at least see when things go wrong. Happy to put a hex dump with string escape ("%x (%q)") back if you want it?

Reply:
Yes, please revert.

Reply:
Thanks! Hex dump is a good idea 👍